helios-tools/src/main/java/com/spotify/helios/cli/CliParser.java

```diff
@@ -95,6 +95,7 @@
   private final List<Target> targets;
   private final String username;
   private boolean json;
+  private boolean strictStart;
```

Comment:
could you add `strictStart` to `WildcardJobCommand` instead of as a global flag for all commands in CliParser? It looks to me like it is only used and needed in `WildcardJobCommand` instances.